pkg/segment/reader/record/rrcreader.go

```diff
@@ -346,7 +346,7 @@ func GetJsonFromAllRrc(allrrc []*utils.RecordResultContainer, esResponse bool, q
 				// it's an async query we're running this function with
 				// len(segmap)=1 because we try to process the data as the
 				// searched complete.
-				log.Infof("qid=%d, GetJsonFromAllRrc: Did not find index for record indentifier %s.", qid, recInden)
+				nodeRes.StoreGlobalError("GetJsonFromAllRrc: Did not find index for record identifier")
```

Comment:
From the comment in the code, I think getting here is expected in some cases. Are you sure we want to upgrade this from an `Info` log to an `Error` log?